Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-getReadConcern.txt

MongoDB\\Collection::getReadConcern()
=====================================

.. meta::

>    :description: Retrieve the read concern for a MongoDB collection using the `getReadConcern()` method.

Reviewer: ```suggestion
   :description: Retrieve the read concern of a collection with the MongoDB PHP Library.
```
